purism/librem/5r4/kernel/6.1.10.nix

```
@@ -0,0 +1,14 @@
+{ buildLinux, fetchFromGitLab, ... }@args:
+buildLinux (args // rec {
+  defconfig = "librem5_defconfig";
+  version = "6.1.10-librem5";
+  modDirVersion = version;
+  src = fetchFromGitLab {
+    domain = "source.puri.sm";
+    owner = "Librem5";
+    repo = "linux";
+    rev = "3767f4e356d2c96fadaebd35ffc13af1eadc84b2";
```

```suggestion
    rev = "${version}pureos1";
```

https://source.puri.sm/Librem5/linux/-/tags/pureos%2F6.1.10pureos1